src/components/GoogleOptions/index.js

```diff
@@ -0,0 +1,66 @@
+import { Tooltip } from 'antd';
+import { AiOutlinePlus} from 'react-icons/ai';
+import { FcManager, FcCalendar, FcCheckmark, FcIdea } from 'react-icons/fc';
+import { urls } from '../../constant/urls';
+import { Container } from "./styles";
+
+const GoogleOptions = () => {
+    return (
+        <Container>
+            <div className='options'>
+                <div>
```

Comment:
Here as well, we have options div already then extra div is not needed, we can use `React fragment` to wrap the code.